src/picongpu/include/plugins/radiation/Radiation.kernel

```diff
@@ -221,11 +221,15 @@ void kernelRadiationParticles(ParBox pb,
          * inside the superCell (anymore)
          */
         while (isValid)
-          {
+        {
             // only threads with particles are running
             if (linearThreadIdx < particlesInFrame)
-              {
+            {
 
+                PMACC_AUTO(par,(*frame)[linearThreadIdx]);
+                // get old and new particle momenta
+                const vector_32 particle_momentumNow = vector_32(par[momentum_]);
```

Comment:
`vector_32`? rly? we does the radiation plugin break the general possibility to allow those attributes being double?